.github/workflows/conda-build.yml

```
@@ -31,12 +31,4 @@ jobs:
         conda config --set channel_priority strict
-        conda build recipe --suppress-variables --override-channels --channel conda-forge --channel bioconda --no-anaconda-upload
-
-    - name: Install the built package and test
-      run: |
-        mamba create --name quantmsutils quantms-utils --use-local
-        conda activate quantmsutils
-        conda list
-        quantmsutilsc --help
-        quantmsutilsc --version
\ No newline at end of file
+        conda build recipe --suppress-variables --override-channels --channel conda-forge --channel bioconda --no-anaconda-upload --output-folder ./
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)